reference/supervisor.md

```diff
@@ -462,6 +462,24 @@ refer to [this section](#wb_supervisor_simulation_reset_physics).
 
 **Name**
 
+**wb\_supervisor\_node\_set\_visibility** - *set the visibility of a node*
+
+{[C++](cpp-api.md#cpp_node)}, {[Java](java-api.md#java_node)}, {[Python](python-api.md#python_node)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_node_set_visibility(WbNodeRef node, WbNodeRef from, bool visible);
+```
+
+**Description**
+
+The `wb_supervisor_node_set_visibility()` function sets the visibility of a node from the specified [Camera](camera.md), [Lidar](lidar.md), [RangeFinder](rangefinder.md) or [Viewpoint](viewpoint.md) node. The `node` argument is the node to hide or show, the `from` argument must be either a [Camera](camera.md), [Lidar](lidar.md), [RangeFinder](rangefinder.md) or [Viewpoint](viewpoint.md) node. The `visible` argument specifies whether the node should be hidden (false) or shown (true). By default, all the nodes are visible. It is relevant to show a node only if it was previously hidden using this function.
```

Comment:
Sorry for the late comment, but I think that it is not fully clear that this function is about the visibility in the resulting rendered image.
Even if it doesn't apply for the Lidar, I would at least mention once the word ``image`` and ``3d scene`` (for the Viewpoint).